Amplify/DevMenu/Trigger/LongPressGestureRecognizer.swift

```diff
@@ -47,13 +47,5 @@ class LongPressGestureRecognizer: NSObject, TriggerRecognizer, UIGestureRecogniz
         uiWindow?.addGestureRecognizer(recognizer)
         recognizer.delegate = self
     }
-
-    deinit {
```

Comment:
yes, it's also not possible to remove the gesture from the window without an async context.  I'd like to propose marking DevMenu for deprecation outside of this PR since it is only documented for V1